documentation/staging/content/userguide/managing-domains/domain-lifecycle/startup.md

```diff
@@ -227,6 +230,8 @@ the operator will not override the environment variable based on the shutdown co
 | `SHUTDOWN_TYPE` | `Graceful` | `Graceful` or `Forced` |
 | `SHUTDOWN_TIMEOUT` | 30 | Whole number in seconds where 0 means no timeout |
 | `SHUTDOWN_IGNORE_SESSIONS` | `false` | Boolean indicating if active sessions should be ignored; only applicable if shutdown is graceful |
+| `SHUTDOWN_WAIT_FOR_ALL_SESSIONS` | `false` | `true` to wait for all HTTP sessions during inflight work handling; `false` to wait for non-persisted HTTP sessions only ; only applicable if shutdown is graceful |
```

Comment:
How does SHUTDOWN_WAIT_FOR_ALL_SESSIONS interact with SHUTDOWN_IGNORE_SESSIONS?

Reply:
The 'ignoreSessions' property takes precedence.  If 'ignoreSessions=true' then 'waitForAllSessions' is ignored.  If 'ignoreSessions=false' and  'waitForAllSessions=true' then shutdown will wait till all sessions are complete before shutdown.  If 'ignoreSessions=false' and  'waitForAllSessions=false' then shutdown will only wait for non-persisted sessions to complete.  I'll try and update the description here to try and make it more clear.  I don't even think out docs point this out as I cut-pasted the descriptions I've given from our docs.

Reply:
I've added a note to try and describe the parameter precedence, please review and let me what you think.

Reply:
Looks good!

Reply:
Looks good.